crates/bifrost/src/providers/local_loglet/read_stream.rs

```diff
@@ -84,7 +84,6 @@ impl LocalLogletReadStream {
         // In summary, if loglet reader started before a trim point and data is readable, we should
         // continue reading them. It's the responsibility of the upper layer to decide on a sane
         // value of _from_offset_.
-        read_opts.set_ignore_range_deletions(true);
```

Comment:
This option has been deprecated upstream since 10.2.1 but the Rust bindings are only reflecting this now; the documentation states:

> deprecated in RocksDB 10.2.1: no performance impact if DeleteRange is not used

See: https://github.com/facebook/rocksdb/releases/tag/v10.2.1